SUPPORTED_SCENARIOS.md

```diff
@@ -8,6 +8,8 @@ The goal of this document is to have a single source of information  which scena
 * grub2-efi
   * only for UEFI boot
   * only with GPT (see [bug](https://bugzilla.novell.com/show_bug.cgi?id=889733#c8))
+* systemd-boot
+  * only for UEFI boot
```

Comment:
Does it mean it should work for every UEFI on all architectures like ppc or aarch?

Reply:
PPC does not support UEFI, therefore it should be out of the picture.
For aarch64, while in theory it should be possible, AFAICS it's currently completely untested in the PoC for openSUSE MicroOS (https://build.opensuse.org/project/show/devel:microos:systemd-boot).

IMHO, we should focus on x86_64 UEFI only, but this needs some additional input from the Micro OS side.

Reply:
I have restricted it